src/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -72,21 +72,28 @@ As a networking-heavy protocol, `ChainSync` makes heavy use of `libp2p`. In part
   The set of peers we initially connect to may completely dominate our awareness of other peers, and therefore all state.
   We use a union of `PeerDiscovery` protocols as each by itself is not secure or appropriate for users' threat models.
   The union of these provides a pragmatic and effective solution.
+  Discovery protocols marked as **required** MUST be included in implementations and will be provided by implementation teams.  Protocols marked as **optional** MAY be provided by implementation teams but can be built independently by third parties to augment bootstrap security.
+- **`libp2p.DataTransfer` a family of protocols for transfering data**
+  Filecoin Nodes must run `libp2p.Graphsync`.
 
 More concretely, we use these protocols:
```

Comment:
For all of the below, it may be helpful to also indicate whether or not they are / are not specced and whether or not we intend to spec them? (ie set expectations that reader likely has to do own research for the optional ones and link to specs of the required ones)